eth/backend.go

```diff
@@ -481,6 +481,7 @@ func (s *Ethereum) EthVersion() int                    { return int(s.protocolMa
 func (s *Ethereum) NetVersion() uint64                 { return s.networkID }
 func (s *Ethereum) Downloader() *downloader.Downloader { return s.protocolManager.downloader }
 func (s *Ethereum) Synced() bool                       { return atomic.LoadUint32(&s.protocolManager.acceptTxs) == 1 }
+func (s *Ethereum) IsLesServer() bool                  { return s.lesServer != nil }
```

Comment:
I'd rather we don't add a method here. If you use `admin.nodeInfo` (or the equivalent call against the Eth object), you should be able to derive this whether `eth`, `les` or both protocols are advertised.